src/components/TimelineNav/TimelineNav.tsx

```diff
@@ -342,18 +340,23 @@ export const TimelineNav = ({
    * timelinNavItemRefs array.
    */
   const onBack = () => {
-    /**
-     * Return focus to the button that was clicked to open the current panel;
-     * activeIndexState holds the index of the last nav item selected in the
-     * timelinNavItemRefs array.
-     */
-    timelineNavItemRefs[activeIndexState].current?.focus();
-    /**
-     * Because the 'Back' button's onFocus listener removes the selected nav
-     * item from the tab order, we need to manually insert it back into the tab
-     * order by setting its tabIndex back to "0".
-     */
-    timelineNavItemRefs[activeIndexState].current.tabIndex = '0';
+    const element = timelineNavItemRefs[activeIndexState].current;
+
+    if (element) {
+      /**
+       * Return focus to the button that was clicked to open the current panel;
+       * activeIndexState holds the index of the last nav item selected in the
+       * timelinNavItemRefs array.
+       */
+      element.focus();
+      /**
+       * Because the 'Back' button's onFocus listener removes the selected nav
+       * item from the tab order, we need to manually insert it back into the tab
+       * order by setting its tabIndex back to "0".
+       */
+      element.tabIndex = 0;
```

Comment:
Note the change from `'0'` to `0`. Apparently we're supposed to use numbers, not strings, but before the ref pointed to an `any`.